pysma/__init__.py

```diff
@@ -316,10 +328,14 @@ async def read(self, sensors):
 
         return True
 
-    async def read_logger(self, sensors, start, end):
+    async def read_logger(self, sensors=None, start=None, end=None):
         """Read a logging key and return the results."""
-        payload = {"destDev": [], "key": int(sensors[0].key), "tStart": start, "tEnd": end}
-        result_body = await self._read_body(URL_LOGGER, payload)
+        if self._new_session_data is None:
+            payload = {"destDev": [], "key": []}
+            result_body = await self._read_body(URL_DASH_LOGGER, payload)
```

Comment:
Does the Dashboard logger not support time ranges? Or is it just a simple view?

Reply:
Actually I'm not sure. I checked in DevTools. The "key' in request to /dyn/getDashLogger.json is empty. That's why I use empty key. Maybe time range is also working.

Reply:
Empty key is probably ok if you just need a dashboard view